ioctl/util/crypt.go

```diff
@@ -60,18 +50,12 @@ func decryptAES(key, data []byte) ([]byte, error) {
 	return data, nil
 }
 
-// EncryptString Takes two string, plainText and keyString.
-// plainText is the text that needs to be encrypted by keyString.
+// Encrypt Takes two string, plainText and key.
+// plainText is the text that needs to be encrypted by key.
 // The function will output the resulting crypto text and an error variable.
-func EncryptString(plainText string, keyString string) (cipherTextString string, err error) {
-
-	key := hashTo32Bytes(keyString)
-	encrypted, err := encryptAES(key, []byte(plainText))
-	if err != nil {
-		return "", err
-	}
+func Encrypt(plainText, key []byte) (cipherText []byte, err error) {
 
-	return base64.URLEncoding.EncodeToString(encrypted), nil
+	return encryptAES(HashSHA256(key), plainText)
```

Comment:
it is actually using HashSHA256(key) as encryption key
suggest to move HashSHA256 outside, that is, call the func like:
```
Encrypt( plaintext, HashSHA256(key) )
```
and here
```
return encryptAES(key, plainText)
```